cylc/flow/etc/tutorial/consolidation-tutorial/flow.cylc

```diff
@@ -100,3 +100,5 @@
             RESOLUTION = 0.2
             # The area to generate forecasts for (lng1, lat1, lng2, lat2)
             DOMAIN = -12,48,5,61  # Do not change!
+
+{% include 'etc/python-job.settings' %}
```

Comment:
Shebang needed

Reply:
This file has been replaced by a symlink to a file which _does_ have the required shebang.

Reply:
But the flow.cylc files will need the Jinja2 shebang